src/store/links.js

```diff
@@ -0,0 +1,30 @@
+const state = {
+  /**
+   * Private link path in case user needs to log in first
+   * @return {string} path to the file/folder
+   */
+  privateLinkPath: ''
```

Comment:
rename to privateLinkUrlPath or similar to avoid confusion with OC internal path ?

Reply:
Adjusted.